app/react-native/src/bin/storybook-start.js

```diff
@@ -41,8 +41,17 @@ server.listen(...listenAddr, err => {
 });
 
 if (!program.skipPackager) {
-  const projectRoots = configDir === projectDir ? [configDir] : [configDir, projectDir];
+  let symlinks = []
+  
+  try{
+    const findSymlinksPaths = require('react-native/local-cli/util/findSymlinksPaths'); // eslint-disable-line global-require
+    symlinks = findSymlinksPaths(path.join(projectDir, 'node_modules'), [projectDir]);
+  }catch(e){
+   console.warn(`Unable to load findSymlinksPaths: ${e.message}`); 
+  }
 
+  const projectRoots = (configDir === projectDir ? [configDir] : [configDir, projectDir]).concat(symlinks);
```

Comment:
I think it'd be a good option somewhere to be able to specify a different project root through `storybook-start` for those who have components and assets that need to be included from outside of the storybook root project.